css-modules/side-container.less

```
@@ -0,0 +1,79 @@
+// General styling of tabs. If it ever gets extracted, these styles should go to common component.
+@outlineColor: #797979;
+@outlineWithoutBottom: 0 -1px 0 @outlineColor, -1px 0 0 @outlineColor, 1px 0 0 @outlineColor;
+
+.tabs {
+  height: 100%;
+  margin-left: 1px;
+}
+
+.tabList {
+  padding-top: 6px;
+  border-bottom: none;
+  margin: 0;
+}
+
+.tab {
+  cursor: pointer;
+  display: inline-block;
+  position: relative;
+  list-style: none;
+  margin-bottom: -1px;
+
+  border-style: solid;
+  border-top-left-radius: 9px;
+  border-top-right-radius: 9px;
```

I would generally write this as
```suggestion
  border-radius: 9px 9px 0 0;
```

---

Updated 👍